pkg/data/cpu/cpu_test.go

```diff
@@ -0,0 +1,189 @@
+package cpu
+
+import (
+	"testing"
+
+	"github.com/openshift/osd-network-verifier/pkg/helpers"
+)
+
+// TestArchitecture_Comparable simply forces the compiler to confirm that the Architecture type
+// is comparable. If not (e.g, because a non-comparable field was added to the struct type), this
+// test will fail to compile
+func TestArchitecture_Comparable(t *testing.T) {
+	if ArchARM != ArchX86 {
+		return
+	}
+}
+
+func TestArchitecture_DefaultInstanceType(t *testing.T) {
+	tests := []struct {
+		name         string
+		arch         Architecture
+		platformType string
+		want         string
+		wantErr      bool
+	}{
+		{
```

Comment:
Is there any way we could get a test case to make sure we throw an error on invalid arch?

Reply:
See the test case on lines 123-131

Reply:
I see those, but I still think we should have a test case to ensure that `DefaultInstanceType` catches and returns an error for that case. This would be getting into the block on lines 53-55 in `cpu.go`

Reply:
Thanks for clarifying; added it